kbase-extension/static/kbase/js/widgets/appWidgets2/paramResolver.js

```diff
@@ -1,6 +1,111 @@
-define(['bluebird', 'require'], (Promise, require) => {
+define([
+    'bluebird',
+    'require',
+    './input/checkboxInput',
+    './input/customInput',
+    './input/customSubdataInput',
+    './input/dynamicDropdownInput',
+    './input/fileInput',
+    './input/floatInput',
+    './input/intInput',
+    './input/newObjectInput',
+    './input/select2ObjectInput',
+    './input/selectInput',
+    './input/subdataInput',
+    './input/taxonomyRefInput',
+    './input/textareaInput',
+    './input/textInput',
+    './input/toggleButtonInput',
+    './input/undefinedInput',
+    './view/checkboxView',
+    './view/customView',
+    './view/customSubdataView',
+    './view/dynamicDropdownView',
+    './view/fileView',
+    './view/floatView',
+    './view/intView',
+    './view/newObjectView',
+    './view/select2ObjectView',
+    './view/selectView',
+    './view/subdataView',
+    './view/taxonomyRefView',
+    './view/textareaView',
+    './view/textView',
+    './view/toggleButtonView',
+    './view/undefinedView',
+], (
+    Promise,
+    require,
+    checkboxInput,
+    customInput,
+    customSubdataInput,
+    dynamicDropdownInput,
+    fileInput,
+    floatInput,
+    intInput,
+    newObjectInput,
+    select2ObjectInput,
+    selectInput,
+    subdataInput,
+    taxonomyRefInput,
+    textareaInput,
+    textInput,
+    toggleButtonInput,
+    undefinedInput,
+    checkboxView,
+    customView,
+    customSubdataView,
+    dynamicDropdownView,
+    fileView,
+    floatView,
+    intView,
+    newObjectView,
+    select2ObjectView,
+    selectView,
+    subdataView,
+    taxonomyRefView,
+    textareaView,
+    textView,
+    toggleButtonView,
+    undefinedView
+) => {
     'use strict';
 
+    const inputModules = {
```

Comment:
this is fairly ugly, but I didn't want to use `eval` to convert a string into a module name